transformer_lens/HookedTransformerConfig.py

```diff
@@ -116,6 +116,8 @@ class HookedTransformerConfig:
             the [scaling laws paper](https://arxiv.org/pdf/2001.08361.pdf) found
             that that was a more meaningful number. Ignoring biases and layer
             norms, for convenience)
+        hook_tokens (bool): If true, will add a hook at tokens (useful for implementing
```

Comment:
-> "will add a hook point on the token input to HookedTransformer.forward, which lets you cache or intervene on the tokens" maybe?